addons/medical_treatment/XEH_postInit.sqf

```diff
@@ -21,3 +21,60 @@ if (isServer) then {
 [QGVAR(actionCheckPulseLocal), FUNC(actionCheckPulseLocal)] call CBA_fnc_addEventHandler;
 [QGVAR(actionCheckBloodPressureLocal), FUNC(actionCheckBloodPressureLocal)] call CBA_fnc_addEventHandler;
 [QGVAR(actionPlaceInBodyBag), FUNC(actionPlaceInBodyBag)] call CBA_fnc_addEventHandler;
+
+// Notification lets the player known if they're being treated
+[QGVAR(initiated), {
+    params ["_caller", "_target"];
+
+    // Notifications don't make sense for self-healing or when dead
+    if (_target != ACE_player || {_target == _caller} || {!alive _target}) exitWith {};
+
+    // Can't tell who is providing aid when you're unconscious
+    private _name = if IS_UNCONSCIOUS(_target) then {
+        LLSTRING(UnknownPerson);
+    } else {
+        name _caller
+    };
+
+    QGVAR(notification) cutText [format [LLSTRING(Notification), _name], "PLAIN DOWN"];
+
+    // We track who is currently treating the unit
+    private _responders = _target getVariable [QGVAR(responders), []];
+    _responders pushBack _caller;
```

Comment:
This is lacking a sanity check against duplicate entries. Also another edge case that should almost never happen, but may be nice to handle here.

Reply:
`pushBackUnique` nicely fixes this